cosmosc2/stream_shared.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+# -*- coding: latin-1 -*-
+"""
+stream_api.py
+"""
+
+# Copyright 2022 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+import json
+
+from cosmosc2.authorization import CosmosAuthorization, generate_auth
+from cosmosc2.environment import COSMOS_SCOPE
+from cosmosc2.stream import CosmosAsyncStream
+
+
+class CosmosAsyncClient:
+    def __init__(
+        self,
+        stream: CosmosAsyncStream,
+        scope: str = COSMOS_SCOPE,
+        auth: CosmosAuthorization = None,
+    ) -> None:
```

Comment:
What does -> None mean?

Reply:
returns nothing, way you can hint at what is being returned